crates/core/src/operations/connect.rs

```diff
@@ -181,6 +181,9 @@ pub(crate) trait RelayContext {
         visited: &[PeerKeyLocation],
         recency: &HashMap<PeerId, Instant>,
     ) -> Option<PeerKeyLocation>;
+
+    /// Whether the acceptance should be treated as a short-lived transient link.
+    fn transient_hint(&self, acceptor: &PeerKeyLocation, joiner: &PeerKeyLocation) -> bool;
```

Comment:
is this for downstreams during connect? if so clarify in the documentation, where/how is this used, why is needed, etc.

Reply:
Clarified in code: this per-node ConnectForwardEstimator tracks downstream Connect forwarding outcomes (success/fail) so a node can bias future Connect forwards toward peers likely to accept/complete when capacity is scarce. Added doc comments on the estimator explaining what it learns and why.